modules/installation-configuration-parameters.adoc

```diff
@@ -677,9 +677,12 @@ endif::openshift-origin[]
 |publish:
 |How to publish or expose the user-facing endpoints of your cluster, such as the Kubernetes API, OpenShift routes.
 |
-ifdef::aws,azure,gcp,ibm-cloud[]
+ifdef::aws,gcp,ibm-cloud[]
 `Internal` or `External`. To deploy a private cluster, which cannot be accessed from the internet, set `publish` to `Internal`. The default value is `External`.
 endif::[]
+ifdef::azure[]
+`Internal`, `External`, or `Mixed`. To deploy a private cluster, which cannot be accessed from the internet, set `publish` to `Internal`. The default value is `External`. To deploy a cluster where the API and the ingress server have different publishing strategies, set `publish` to `Mixed` and use the `operatorPublishingStrategy` parameter.
```

Comment:
I'm guessing because of the title of the table we don't need nouns after `Internal`, `External` etc?

Reply:
Right, these are under the "Values" heading and so they aren't full sentences.